src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir-Heavy", sans-serif;
+}
+
+.container--l {
+  padding: 20px 8px;
+}
+
+.container--s {
+  padding: 0 8px;
+}
+
+.form {
+  position: relative;
+  display: flex;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-general {
+  width: 100%;
+  outline: none;
+  border: none;
+  font-family: inherit;
+  color: #3d4e61;
+}
+
+.button-general {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  height: 100%;
+  background: none;
+  cursor: pointer;
+}
+
+.input-general:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input-general:focus {
```

Comment:
text should become bolder only on focus
<img width="314" alt="image" src="https://user-images.githubusercontent.com/66749919/216399442-0aaaaa0a-3176-4c3e-9f50-2c930fcf754c.png">